### ISSUE TITLE
slack import fails due to file size greater than 0

### ISSUE THREAD

**Issue by kneufeld** [RocketChat/Rocket.Chat#9577]
### Server Setup Information:

* Version of Rocket.Chat Server: 0.61.0
* Operating System: 4.4.0-112-generic
* Deployment Method(snap/docker/tar/etc): docker
* Number of Running Instances: 1
* DB Replicaset Oplog: disabled
* Node Version: v8.9.3

### Steps to Reproduce:

1. try to import a slack zip file
2. pick a single channel (no users)

### Expected behavior:

Messages get imported

### Actual behavior:

There are errors in the log and no messages are imported.

### Relevant logs:

```
2/1/2018 5:06:11 PMException in callback of async function: Error: File exceeds allowed size of 0 B. [error-file-too-large]
2/1/2018 5:06:11 PM    at Filter.validateFileUpload [as onCheck] (/app/bundle/programs/server/packages/rocketchat_file-upload.js:119:10)
2/1/2018 5:06:11 PM    at Filter.check (/app/bundle/programs/server/packages/jalik_ufs.js:558:57)
2/1/2018 5:06:11 PM    at FileUploadClass.insert (/app/bundle/programs/server/packages/rocketchat_file-upload.js:587:11)
2/1/2018 5:06:11 PM    at res.on.Meteor.bindEnvironment (/app/bundle/programs/server/packages/rocketchat_importer.js:366:15)
2/1/2018 5:06:11 PM    at runWithEnvironment (packages/meteor.js:1240:24)
```

**Comment by kneufeld** [comment]
The import process changes the `File Upload`.`Maximum File Upload Size` to 0 before starting the import and then seems to complain when it's 0.

**Comment by kneufeld** [comment]
Rocket.Chat/packages/rocketchat-file-upload/lib/FileUpload.js

`if` on line 33 will always hit as you set `maxFileSize` to 0 at the top of the file.

**Comment by graywolf336** [comment]
Relevant Importer commit: https://github.com/RocketChat/Rocket.Chat/commit/ca5b24f53eab02d48e02dee4d0e1ad17e6e2e1eb

**Comment by geekgonecrazy** [comment]
Seeing a few more cases of this.

**Comment by twardnw** [comment]
Yes, I am being stuck by this currently, logs look the same as those above, but happy to provide if you would like

**Comment by tjhorner** [comment]
I am experiencing this on version 0.62.2, except it's saying `-1` instead of `0`. 31ead9ad92867f19b00d0e1256a423773ce2fc2e says that -1 should be unlimited, but it seems it's taking it literally as -1.

![image](https://user-images.githubusercontent.com/2646487/37249387-332f2538-249b-11e8-905f-9ec1854e955d.png)

This can be worked around by setting the max upload size to something huge mid-import.

**Comment by graywolf336** [comment]
@tjhorner can you open a new issue for that? It should be working as it worked locally for me, but I will have to test it more.
